src/main/scala/intent/core/internal.scala

```diff
@@ -13,6 +13,10 @@ trait TestSupport extends FormatterGivens with EqGivens with ExpectGivens
 trait IntentStructure:
   private[intent] def allTestCases: Seq[ITestCase]
 
+  case class IgnoredTestCase(nameParts: Seq[String]) extends ITestCase:
```

Comment:
Does this really need to be nested below `IntentStructure`?

Reply:
No reason, I'll extract it